packages/nr-metadata/nr_metadata/ui/semantic-ui/js/nr/search/components/EmptyResultsElement.jsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import { Grid, Header, Button, Icon, Segment } from "semantic-ui-react";
+import { i18next } from "@translations/i18next";
+
+export const EmptyResultsElement = ({
+  queryString,
+  searchPath,
+  resetQuery,
+}) => {
+  return (
+    <Grid>
+      <Grid.Row centered>
+        <Grid.Column width={12} textAlign="center">
+          <Header as="h2">
+            {i18next.t("We couldn't find any matches for ")}
+            {(queryString && `'${queryString}'`) || i18next.t("your search")}
+          </Header>
+        </Grid.Column>
+      </Grid.Row>
+      <Grid.Row centered>
+        <Grid.Column width={8} textAlign="center">
+          <Button primary onClick={resetQuery}>
+            <Icon name="search" />
+            {i18next.t("Start over")}
+          </Button>
+        </Grid.Column>
+      </Grid.Row>
+      <Grid.Row centered>
+        <Grid.Column width={12}>
+          <Segment secondary padded size="large">
+            <Header as="h3" size="small">
+              {i18next.t("ProTip")}!
+            </Header>
+            <p>
+              <a
+                href={`${searchPath}?q=metadata.publication_date:[2017-01-01 TO *]`}
```

Comment:
Is there a metadata.publication_date on nr_metadata model? Otherwise, this link would break search / just not be working

Reply:
Yep we either should fallback to upstream empty results, or just remove broken links from here (and leave just the generic "start over" part)

Reply:
@Ducica can you please update the component here? :)

Reply:
@mirekys I am using regular empty component in NR docs and this one I was thinking we can revisit once we have that help page as well?

Reply:
@Ducica agreed, will you put up a linear issue please? you can then mark it as resolved here :)